draft-ietf-rats-architecture.md

```diff
@@ -133,6 +123,10 @@ Attester:
 
 : An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
 
+Claim:
```

Comment:
One minor question: if we define the term Claim, do we need to further change the 'claim' now in the whole doc to capitalized 'Claim'?